queries.sql

```diff
@@ -8,3 +8,55 @@ SELECT name,escape_attempts FROM animals WHERE  weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals  WHERE weight_kg >=10.4 AND  weight_kg <=17.3;
+
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name ILIKE '%mon%';
+
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL ;
+
+COMMIT;
+
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+
+BEGIN;
+DELETE FROM animals
+WHERE
+date_of_birth > '2022-01-01';
+
+SAVEPOINT first;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO first;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT; 
+
```

Comment:
- [ ] The screenshot for this transaction isn't correct. The final screenshot that you have shows negative values for the `weight_kg` columns of the records in the `animals` table. Kindly attach the correct screenshot. 👍 💻